enginetest/queries/queries.go

```diff
@@ -8320,6 +8320,10 @@ type QueryErrorTest struct {
 }
 
 var ErrorQueries = []QueryErrorTest{
+	//{
```

Comment:
Better to put this in BrokenErrorQueries so it generates a skip, too easy to ignore as a comment